docs/source/notes/autograd.rst

```diff
@@ -210,3 +210,116 @@ No thread safety on C++ hooks
 Autograd relies on the user to write thread safe C++ hooks. If you want the hook
 to be correctly applied in multithreading environment, you will need to write
 proper thread locking code to ensure the hooks are thread safe.
+
+Autograd for Complex Numbers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**What notion of complex derivative does PyTorch use?**
+*******************************************************
+
+PyTorch follows `JAX's <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
+convention for autograd for Complex Numbers.
+
+Suppose we have a function :math:`F: ℂ → ℂ` which we can decompose into functions u and v
+which compute the real and imaginary parts of the function:
+
+    .. code::
+
+        def F(z):
+            x, y = real(z), imag(z)
+            return u(x, y) + v(x, y) * 1j
+
+where :math:`1j` is a unit imaginary number.
+
+We define the JVP for function :math:`F` at :math:`(x, y)` applied to a tangent
+vector :math:`c+dj \in C` as:
```

Comment:
it's probably better to explicitly say that this is Python pseudocode. For example, you couldn't actually use `*` in a real PyTorch program as that would give you pointwise multiplication, not matrix product. And the bracket syntax means something else in Python, that is also not intended here either.

Or even better, make this real code using the real PyTorch operations. Then you don't have to explain the pseudocode syntax.

Reply:
I updated this section to just use math notation instead. It's simpler to read and avoids the confusion